src/firmware/shared/util.c

```diff
@@ -5,7 +5,7 @@
 float fmax_of_array(float array[], unsigned size)
 {
     unsigned i;
-    float max_value;
+    float max_value = 0.0f;
```

Comment:
Note: I built with optimizations and compilation failed because this value is "possibly uninitialized", which is true if the `size=0`. That's why these changes exist (not sure why this wasn't caught with a normal fastbuild).